specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/examples/ExpressRouteConnectionCreate.json

```diff
@@ -0,0 +1,50 @@
+{
+    "parameters": {
+        "expressRouteGatewayName": "expressRouteGatewayName",
+        "resourceGroupName": "resourceGroupName",
+        "api-version": "2018-08-01",
+        "subscriptionId": "subid",
+        "connectionName": "connectionName",
+        "putExpressRouteConnectionParameters": {
+            "id": "/subscriptions/subid/resourceGroups/resourceGroupName/providers/Microsoft.Network/expressRouteGateways/expressRouteGatewayName/expressRouteConnections/connectionName",
+            "name": "connectionName",
+            "properties": {
+                "routingWeight": 2,
+                "authorizationKey": "authorizationKey",
+                "expressRouteCircuitPeering": {
+                    "id": "/subscriptions/subid/resourceGroups/resourceGroupName/providers/Microsoft.Network/expressRouteCircuits/circuitName/peerings/AzurePrivatePeering"
```

Comment:
Since you are specifying an auth key here, you should specify circuit sub id as sub2